server/types.ts

```diff
@@ -424,9 +424,6 @@ export type ShareEvent = BaseEvent<Share> & {
   modelId: string;
   documentId: string;
   collectionId?: string;
-  data: {
-    name: string;
```

Comment:
Removed `name` data as it wasn't used anywhere in the background jobs.

Reply:
sg, as long as it's in the changeset we should be removing usage of `data` whereever possible.